src/features/MoviesBrowser/MoviePage/useMovieData.js

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { API_KEY } from "../../../codesAPI";
+
+const useMovieData = (movieId) => {
+  const [movieData, setMovieData] = useState({
+    status: "loading",
+    movie: [],
+    credits: [],
+  });
+
+  const MOVIE_API_URL = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}`;
+  const CREDITS_API_URL = `https://api.themoviedb.org/3/movie/${movieId}/credits?language=en-US&api_key=${API_KEY}`;
```

Comment:
I thought I could get the url from one place

Reply:
I'm not sure what do you mean by that.. can you explain? :D

Reply:
so that these two URLs can be transferred to codesAPI 😊

Reply:
Sure thing, already corrected 😎